src/System.Management.Automation/engine/hostifaces/PSCommand.cs

```diff
@@ -85,19 +85,19 @@ internal PSCommand(Command command)
         /// <exception cref="ArgumentNullException">
         /// cmdlet is null.
         /// </exception>
-        public PSCommand AddCommand(string command)
+        public PSCommand AddCommand(string cmdlet)
```

Comment:
[Changing this is a breaking change](https://stackoverflow.com/a/8018728)

Reply:
Noted. I will exclude this breaking change.